app/lib/submission_builder/ty_2021/adv_ctc_irs1040.rb

```diff
@@ -42,33 +42,42 @@ def document
               dependent_xml(xml, dependent)
```

Comment:
The name of this file (adv_ctc_irs_1040.rb) reflected the fact that last year's revenue procedure was to claim advanced ctc money. I think you can change this to under_standard_deduction_irs_1040.rb or lapsed_filer_irs_1040.rb and it would better reflect what this is for this year.

Reply:
Renamed! :)